test/integration/test-amp-sidebar.js

```diff
@@ -0,0 +1,119 @@
+/**
```

Comment:
Tests for extensions should be in their respective extension folder, so let's move this under `extensions/amp-sidebar/0.1`. The exact path is up to your best judgment, but it could be something like `extensions/amp-sidebar/0.1/test/test-amp-sidebar-integration.js` or `extensions/amp-sidebar/0.1/test/integration/test-amp-sidebar.js`

Reply:
I'm not entirely sure about this because `amp-img` and `amp-carousel` both have their integration tests in the `integration` folder.

Reply:
Ah yeah it doesn't look like there's a clear pattern. Also `bind` has integration tests in `test/integration` and in `extensions/amp-bind/0.1/test/integration`. I'm not sure what the difference is. @choumx do you have any guidance?

Reply:
Probably better to put it in the extension folder. Note that integration tests must follow specific filename conventions to be run on SauceLabs: https://github.com/ampproject/amphtml/blob/adc00868895f6380a357171bc714269b79fb561e/build-system/config.js#L92-L96

This means it must be under `extensions/amp-sidebar/0.1/test/integration/`. I made this error with amp-bind integration tests which is why they live in `test/integration`. :)

Reply:
Got it. Thanks!
- [ ] Move to `extensions/amp-sidebar/0.1/test/integration/`